src/main/java/com/github/wovnio/wovnjava/CustomDomainLanguageSerializer.java

```diff
@@ -0,0 +1,55 @@
+package com.github.wovnio.wovnjava;
+
+import java.util.ArrayList;
+import java.net.MalformedURLException;
+import java.net.URL;
+
+class CustomDomainLanguageSerializer {
+    private CustomDomainLanguageSerializer() {}
+
+    public static ArrayList<CustomDomainLanguage> deserializeFilterConfig(String rawCustomDomainLanguages) throws ConfigurationError {
+        ArrayList<CustomDomainLanguage> customDomainLanguageList = new ArrayList<CustomDomainLanguage>();
+
+        for (String rawPair : rawCustomDomainLanguages.split(",")) {
+            if (rawPair == null || rawPair.isEmpty()) {
+                continue;
+            }
+            String[] splitPair = rawPair.split(":");
+            if (splitPair.length != 2) {
+                throw new ConfigurationError("Invalid configuration format for \"customDomainLangs\": " + rawCustomDomainLanguages);
+            }
+
+            Lang lang = Lang.get(splitPair[1].trim());
+            if (lang == null) {
+                throw new ConfigurationError("Invalid target language for \"customDomainLangs\", each value must match a supported language code.");
+            }
+
+            String hostAndPath = splitPair[0].trim();
+            try {
+                URL url = new URL("http://" + hostAndPath);
+                customDomainLanguageList.add(new CustomDomainLanguage(url.getHost(), removeTrailingSlash(url.getPath()), lang));
+            }
+            catch (MalformedURLException e) {
+                throw new ConfigurationError("MalformedURLException when parsing \"customDomainLangs\": " + e.getMessage());
+            }
+        }
+
+        return customDomainLanguageList;
+    }
+
+    public static String serializeToJson(CustomDomainLanguages customDomainLanguages) {
+        if (customDomainLanguages == null) {
+            return "";
+        }
+
+        ArrayList<String> items = new ArrayList<String>();
+        for (CustomDomainLanguage cdl : customDomainLanguages.customDomainLanguageList) {
+            items.add(cdl.host + cdl.path + "/\":\"" + cdl.lang.code);
+        }
+        return "{\"" + String.join("\",\"", items) + "\"}";
```

Comment:
I was kind of expecting there to be a standard json serializer in Java, but since this isn't too complicated it's _probably_ OK to manually do it here.

Reply:
Yeah that's what I was thinking as well, and I looked for a good non-manual solution. But I believe it would require us to add another dependency to our `pom.xml`, which feels unnecessary for such a simple serialization. I think it's ok to keep it like this.